### PR TITLE
Add softplus and softsign

### PR DESCRIPTION
**Pull request by Shashi456**
We need these operators for other loss functions like kullback-leibler divergence and logcosh.

One small doubt, The comments for softplus and softsign both mention gradients as 
`  gradients: The backpropagated gradients to the corresponding softplus operation.`

So am i getting the logic right by passing `gradients: v`?

Tests pass.

### CODE REVIEW

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -766,6 +766,38 @@ internal func _vjpSigmoid<T: TensorFlowFloatingPoint>(
     (sigmoid(x), { v in Raw.sigmoidGrad(x, dy: v) })
 }
 
+/// Computes the softplus of the specified tensor.
+// Specifically, computes `log(exp(features) + 1)`.
```
Comment (saeta) [inline Sources/TensorFlow/Operators/Math.swift]:
```suggestion
/// Specifically, computes `log(exp(features) + 1)`.
```

File: Sources/TensorFlow/Operators/Math.swift [hunk 2]
```diff
@@ -766,6 +766,38 @@ internal func _vjpSigmoid<T: TensorFlowFloatingPoint>(
...
+
+@inlinable
+internal func _vjpSoftplus<T: TensorFlowFloatingPoint>(
+    _ features: Tensor<T>
+) -> (Tensor<T>, (Tensor<T>) -> Tensor<T>) {
+    (softplus(features), { v in Raw.softplusGrad(gradients: v, features: features)})
+}
+
+/// Computes the softsign of the specified tensor.
+/// Specifically, computes `features/ (abs(features) + 1)`.
```
Comment (saeta) [inline Sources/TensorFlow/Operators/Math.swift]:
```suggestion
/// Specifically, computes `features / (abs(features) + 1)`.
```

File: Sources/TensorFlow/Operators/Math.swift [hunk 3]
```diff
@@ -766,6 +766,38 @@ internal func _vjpSigmoid<T: TensorFlowFloatingPoint>(
...
+/// Computes the softplus of the specified tensor.
+// Specifically, computes `log(exp(features) + 1)`.
```
Comment (saeta) [inline Sources/TensorFlow/Operators/Math.swift]: Looks like you've copy-pasted unnecessary comments. Please remove! :-)

File: Tests/TensorFlowTests/OperatorTests/MathTests.swift [hunk 4]
```diff
@@ -214,6 +214,24 @@ final class MathOperatorTests: XCTestCase {
...
+    }
+
+    func testSoftsign() {
+      let x = Tensor<Float>([1.0, 4.0, 3.0])
+      let y = softsign(x)
+      let expected = Tensor<Float>([0.5 , 0.8 , 0.75]])
+      XCTAssertEqual(x, expected)
+    }
+
+    func testSoftsign() {
```
Comment (saeta) [inline Tests/TensorFlowTests/OperatorTests/MathTests.swift]: Could you please remove?

**Comment by Shashi456** [comment]
@saeta sorry, I was probably copying them for reference but didn't remove them.

**Comment by eaplatanios** [comment]
@Shashi456 @saeta `softplus` is already added in #148.

**Comment by Shashi456** [comment]
@rxwei should I remove softplus from this ? Also #148 takes the derivative as sigmoid, I know it's right. But is there a good reason to not use softplusGrad given within the bindings?

**Comment by Shashi456** [comment]
@rxwei could you put a build tag on this or are you waiting for #148? In that case I'll remove softplus from here and let this PR go ahead with adding softsign.

**Comment by Shashi456** [comment]
Sorry, made some changes for build errors but did not push them till now.

**Comment by Shashi456** [comment]
@rxwei :P, Can you trigger a build once more?

**Comment by Shashi456** [comment]
@rxwei so the build passes, the error is from `move`, was the toolchain updated for that?

**Comment by Shashi456** [comment]
Requesting a build on this.

File: Sources/TensorFlow/Operators/Math.swift [hunk 5]
```diff
@@ -896,6 +896,36 @@ internal func _vjpSigmoid<T: TensorFlowFloatingPoint>(
     (sigmoid(x), { v in Raw.sigmoidGrad(x, dy: v) })
 }
 
+/// Computes the softplus of the specified tensor element-wise.
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]:
```suggestion
/// Returns the softplus of the specified tensor element-wise.
```
#160

Comment (Shashi456) [reply Sources/TensorFlow/Operators/Math.swift]: Done

**Comment by rxwei** [comment]
Thanks!
